src/vision/tracker.py

```
@@ -0,0 +1,582 @@
+        x, y, w, h = opencv_bbox
+        if w <= 0 or h <= 0 or x < 0 or y < 0 or x + w > width or y + h > height:
+            logger.warning(f"Invalid bounding box: {opencv_bbox}")
+            return False
+
+        # Create new tracker
+        self._tracker = self._create_tracker()
+
+        try:
+            success = self._tracker.init(frame, opencv_bbox)
+        except Exception as e:
+            logger.exception(f"Tracker initialization failed: {e}")
```

Ruff (0.14.10)

Redundant exception object included in `logging.exception` call

(TRY401)